server/src/main/resources/templates/web/passcode.html

```diff
@@ -8,6 +8,19 @@
 <div layout:fragment="page-nav">
```

Comment:
This file's changes can be backed out now, per @cnelson's observation that even CLI users have to go through the login screen too before they can reach this page.